src/server/dflycmd.cc

```diff
@@ -97,6 +97,20 @@ struct TransactionGuard {
 
   Transaction* t;
 };
+
+struct ExpirationGuard {
+  explicit ExpirationGuard(EngineShard* s = EngineShard::tlocal()) : shard(s) {
```

Comment:
we should run SetExpireAllowed on all shards, not only on the current thread. Use shard_set->RunBriefInParallel.

Reply:
thx